### DISCUSSION TITLE
Change the text from On / Off buttons to Enable / Disable in the mod selection window.

### DISCUSSION THREAD

**Discussion by scailman** [voxeltycoon/issues#403] [UI&UX]
I suggest this small change because when it comes to translating the text, in this case into Spanish, it makes more sense to say available than on.
There are differences between the words available / not available, on / off and on / off (Activated / Deactivated) in Spanish and I don't know if in more languages.

Adding a button to activate / deactivate all mods at the top of the mod selection window is more for convenience when selecting a specific mod if you do not want to have all those installed.

**Comment by victorleblais** [comment]
I created a suggestion for the second part of your post:

> "Adding a button to activate / deactivate all mods at the top of the mod selection window is more for convenience when selecting a specific mod if you do not want to have all those installed."

See issue #109
